File: C&C++/MEHUL/Day10_Q1.c
#include<stdio.h>

int main()
{
    int i, j, r, s; 
    
    printf("Enter the range: ");
    scanf("%d", &r);
    
    int fibona[r];
    
    fibona[0] = 0;
    fibona[1] = 1;
    
    for(i=2; i<=r; i++)
    {
        s = fibona[i-2] + fibona[i-1];
        fibona[i] = s;
    }
    
    for(i=0; i<=r; i++)
    {
        printf("%d ", fibona[i]);
    }
}

File: Python/MEHUL/Day10_Q1.py
def calculate_discount(total, discount):
    discount_amount = total * (discount/100)
    final_price = total-discount_amount
    
    return final_price


items = int(input("Enter number of items: "))
discount = int(input("Enter the discount: "))

total = 0

for i in range(0, items):
    price = float(input("Enter the price of each item: "))
    total = total + price
    
final_price = calculate_discount(total, discount)

print(f"Price before discount: {total} \nPrice after discount: {final_price}")
